ci/jenkins/jobs/projects.yaml

```diff
@@ -1147,3 +1147,59 @@
              - timeout:
                  timeout: 600
                  type: absolute
+      - '{name}-{test_name}-for-pull-request':
+            test_name: vm-e2e
+            node: 'antrea-test-node'
```

Comment:
This is the job label we use. You can change it to antrea-vm-node

Reply:
Done